### PR TITLE
[6527] Adjust .clang-format and .clang-tidy (main)

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #6527 for main.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -1,184 +1,203 @@
...
+UseCRLF:         false
+
+#UseTab:          AlignWithSpaces
+UseTab:          Never
+TabWidth:        4
+IndentWidth:     4
+ConstructorInitializerIndentWidth: 4
+
+IndentPPDirectives: AfterHash
+PPIndentWidth:   1
```
Comment (korydraughn) [inline .clang-format:17]: Based on the example in the clang-format docs, I think we need to bump this to at least `2`. One space for indentation can make it difficult to follow the logic.

**Review by korydraughn — COMMENTED** [review]
Looks good. Just one comment to address.

**Comment by SwooshyCueb** [comment]
If there are no more objections, I will \# and cherry-pick into 4-2-stable and 4-3-stable

**Review by korydraughn — APPROVED** [review]
Ship it!

**Comment by korydraughn** [comment]
I'm thinking we use the tab-based version of clang-format and clang-tidy on either the C++ REST API and/or the iRODS CLI.

Those are good projects that will help us tighten up the clang files even more.

Any thoughts?

**Comment by SwooshyCueb** [comment]
I'm down for both

**Comment by trel** [comment]
yes please, and we can learn about the 'store it over here, use it over there' approach as well.

then... we move this file to irods_development_environment as discussed.

**Comment by korydraughn** [comment]
Agreed.

**Review by alanking — APPROVED** [review]

**Comment by korydraughn** [comment]
Is this ready?

If yes, let's add the pound sign and merge it.

**Review by trel — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**